pkg/controllers/rollout/controller.go

```diff
@@ -20,9 +20,10 @@ import (
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog/v2"
-	ctrl "sigs.k8s.io/controller-runtime"
+	controllerRuntime "sigs.k8s.io/controller-runtime"
```

Comment:
By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. 

rename to "runtime" or "controllerruntime"?